content/influxdb/clustered/reference/release-notes/clustered.md

```diff
@@ -36,6 +36,49 @@ spec:
     image: us-docker.pkg.dev/influxdb2-artifacts/clustered/influxdb:20241022-1346953
 ```
 
+### Known Bugs
+
+### `core` service DSN parsing errors
+
+This release has a known bug in the `core` pods with respect to handling of
+options in Postgres DSNs. This bug can be seen in the `core-MMMMMMMMMM-NNNNN`
+logs that look like the following:
+
+```
+2024-11-04T01:00:00.000Z | 3: error returned from database: database "influxdb&options=-c%20search_path=" does not exist
+2024-11-04T01:00:19.000Z | 4: database "influxdb&options=-c%20search_path=" does not exist
+```
+
+The issue here is that entire `influxdb&options=-c%20search_path=` string is
+being interpreted as the database name, owing to incorrect parsing of the
+`POSTGRES_DSN` environment variable.
+
+The workaround to deal with this before a fix makes its way into the next
+release is to use an image override in your AppInstance that looks like:
+
+```
+apiVersion: kubecfg.dev/v1alpha1
+kind: AppInstance
+metadata:
+  name: influxdb
+  namespace: influxdb
+spec:
+  package:
+    image: us-docker.pkg.dev/influxdb2-artifacts/clustered/influxdb:20241024-1354148
+    apiVersion: influxdata.com/v1alpha1
+    spec:
+      images:
+        overrides:
+          - name: 'influxdb2-artifacts/granite/granite'
+            newFQIN: 'us-docker.pkg.dev/influxdb2-artifacts/granite/granite:7acf9ca6e1ad15db80b22cd0bc071acdb561eb51'
```

Comment:
Is this how we would run a local or GKE Clustered instance with a locally-build or not-yet-promoted Granite image as well? If so, this is definitely useful outside of the release notes as well 🙌

Reply:
Yeah! I've done it with IOx back when I was implementing license enforcement.

Although, it's probably not very useful for you to develop a granite image against a released version of Clustered given how fast things change -- for example, if you tried to do this with a Granite build from today's main branch the `accounts` and `core` pods will crash loop because of new required args/envvars (I know because I tried a more recent version of Granite than what's shown here before I went with is image).

It's probably more useful to render Clustered yaml using `kubecfg` (see https://github.com/influxdata/k8s-idpe/tree/master/env/clustered#rendering-yaml-without-the-operator) and use `kubectl` to deploy those to the cluster with whatever Granite changes you want made directly in that repo.